### PR TITLE
fix training

### PR DESCRIPTION
**Pull request by GreatSnoopyMe**
Fix the tests/test_training.py and tests/test_training_utils.py
## Description

Have successfully tested all Node_classification datasets. But still require further optimization on the graph classfication datasets.

### CODE REVIEW

**Comment by jiaqima** [comment]
Hi @StevenGolden1203, could you take a look at this PR? Thx!

File: tests/test_training.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    Else, assert False.
+    Use model GCN to do test training.
+    """
+    args, model_cfg, train_cfg = get_cfg(dataset_name)
+    device = "cpu"
+
+    data = gli.dataloading.get_gli_dataset(args["dataset"], args["task"], 1,
+                                           device=device)
+
+    # check EdgeFeature and multi-modal node features
```
Comment (jiaqima) [inline tests/test_training.py:34]: Please remove commented code.

**Review by StevenGolden1203 — COMMENTED** [review]
Currently we replicate training code in `test_training.py`. If we also copy&paste the training loop for graph classification, the code may become redundant. Even so, this method can be used as a temporary fix.

A better way may be warpping the training procedure into a class: Trainer, in this way `test_training` can import the Trainer and do training.

**Comment by jiaqima** [comment]
> Currently we replicate training code in `test_training.py`. If we also copy&paste the training loop for graph classification, the code may become redundant. Even so, this method can be used as a temporary fix.
> 
> A better way may be warpping the training procedure into a class: Trainer, in this way `test_training` can import the Trainer and do training.

This is a good point. But I think this is good enough for now. We do not need to optimize the testing code too much.

Regarding the implementation, I think the current test code does not check if the dataset is a node classification dataset? We may need something like the following line in `test_kg_training.py`:

https://github.com/Graph-Learning-Benchmarks/gli/blob/c70ec9e70fe75752c15d2436b4cad8dda4bbcee7/tests/test_kg_training.py#L23

@StevenGolden1203, what do you think?

@GreatSnoopyMe, please also fix the two comments above by @StevenGolden1203 and myself.

File: tests/test_training_utils.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
+"""Functions used in test_training."""
```
Comment (jiaqima) [inline tests/test_training_utils.py:1]: Let's rename this file as `training_utils.py` since this file does not contain any tests.

**Comment by StevenGolden1203** [comment]
> > Currently we replicate training code in `test_training.py`. If we also copy&paste the training loop for graph classification, the code may become redundant. Even so, this method can be used as a temporary fix.
> > A better way may be warpping the training procedure into a class: Trainer, in this way `test_training` can import the Trainer and do training.
> 
> This is a good point. But I think this is good enough for now. We do not need to optimize the testing code too much.
> 
> Regarding the implementation, I think the current test code does not check if the dataset is a node classification dataset? We may need something like the following line in `test_kg_training.py`:
> 
> https://github.com/Graph-Learning-Benchmarks/gli/blob/c70ec9e70fe75752c15d2436b4cad8dda4bbcee7/tests/test_kg_training.py#L23
> 
> @StevenGolden1203, what do you think?
> 
> @GreatSnoopyMe, please also fix the two comments above by @StevenGolden1203 and myself.

Then we may rename this file as `test_node_training.py` (@GreatSnoopyMe ), which only test training on node classification dataset. I will create a separate PR for `test_graph_training.py`.

**Comment by jiaqima** [comment]
Please remove the two redundant files so that this PR can be merged.

**Pull request merged**